auth/login/auth-login.component.ts

```diff
@@ -128,6 +130,12 @@ export class AuthLoginComponent {
     }
 
 
+    openPrivacyLink() {
+        window.open(environment.zeta.getPrivacyLink(this.i18n.language), '_blank').focus();
```

Comment:
before executing this you should assert that `getPrivacyLink` is defined. The privacy button may not be able to call this function in that case, but someone else may.
Use the bool flag from above (e. g. `privacyLinkDefined`).

Reply:
i add an if statement  window.open(environment.zeta.getPrivacyLink(..... will be called when privacyLinkDefined ==true